scripts/core/common.yml

```diff
@@ -109,6 +109,26 @@ desc: "Handle of memory object which can either be buffer or image"
 class: $xMem
 name: "$x_mem_handle_t"
 --- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of bindless image"
+class: $xBindlessImages
+name: "$x_exp_image_handle_t"
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of bindless image memory"
+class: $xBindlessImages
+name: "$x_exp_image_mem_handle_t"
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of interop memory"
+class: $xBindlessImages
+name: "$x_exp_interop_mem_handle_t"
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of interop semaphore"
+class: $xBindlessImages
+name: "$x_exp_interop_semaphore_handle_t"
+--- #--------------------------------------------------------------------------
```

Comment:
These should instead be defined at the top of `scripts/core/exp-bindless-images.yml`.

Its not always possible, such as extending existing enums, but the idea is to avoid touching the core yaml files as much as possible when defining an experimental extension.